pkg/auth/providers/activedirectory/activedirectory_client.go

```diff
@@ -115,18 +119,18 @@ func (p *adProvider) RefetchGroupPrincipals(principalID string, secret string) (
 		return nil, err
 	}
 
-	dn := externalID
-
-	logrus.Debugf("LDAP Refetch principals base DN : {%s}", dn)
+	escapedGUID := common.EscapeUUID(externalID)
+	logrus.Debugf("LDAP Refetch principals GUID : {%s}", externalID)
```

Comment:
I am wondering it might be useful to log the DN as well here - an automatic data trace to point out the user in human readable format and also map it to the new GUID?

Reply:
We actually don't have the DN at this point, only the principal (GUID) is passed in here.